api/src/main/java/org/pmiops/workbench/institution/InstitutionServiceImpl.java

```diff
@@ -375,6 +369,22 @@ public boolean validateOperationalUser(DbInstitution institution) {
         .collect(Collectors.toList());
   }
 
+  @Override
+  public List<String> getUserEligabledAccessTiers(DbUser user) {
+    Optional<Institution> institution = getByUser(user);
+    if (!institution.isPresent()) {
+      return new ArrayList<String>();
+    }
+    System.out.println(accessTierDao.findAll());
+    return accessTierDao.findAll().stream()
+        .filter(
+            a ->
+                validateInstitutionalEmail(
+                    institution.get(), user.getContactEmail(), a.getShortName()))
+        .map(DbAccessTier::getShortName)
```

Comment:
since you're only using the ShortName from the AccessTier, you can move the map() before the filter()